qtconsole/console_widget.py

```diff
@@ -1660,10 +1692,7 @@ def _on_flush_pending_stream_timer(self):
         """ Flush the pending stream output and change the
         prompt position appropriately.
         """
```

Comment:
Probably we should update this method docstring to just be something like:

```python
""" Flush the pending stream output."""
```

Reply:
Updated description, and added a bit more detail